frontend/src/app/components/wp-form-group/wp-attribute-group.component.ts

```diff
@@ -28,9 +28,9 @@
 
 import {Component, Injector, Input} from '@angular/core';
 import {I18nService} from 'core-app/modules/common/i18n/i18n.service';
-import {WorkPackageEditFieldGroupComponent} from 'core-components/wp-edit/wp-edit-field/wp-edit-field-group.directive';
 import {FieldDescriptor, GroupDescriptor} from 'core-components/work-packages/wp-single-view/wp-single-view.component';
 import {WorkPackageResource} from 'core-app/modules/hal/resources/work-package-resource';
+import {EditFormComponent} from "core-app/modules/fields/edit/edit-form/edit-form.component";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/e1c1b00c5a6c973508c7538a865d197276aa638c/frontend/src/app/components/wp-form-group/wp-attribute-group.component.ts#L34"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>